### DISCUSSION TITLE
When will Prism.Unity.Maui package come out or what is the replacement for now?

### DISCUSSION THREAD

**Discussion by anzeko123** [PrismLibrary/Prism#2963] [.NET MAUI]
Hi,
As question above, I am asking if there is a time when Prism.Unity package will come to .NET MAUI.
I am migrating from Xamarin to Maui and don't know how to do this other way than using Unity.
I want to register Unity Container I got the `PlatformInitializer` where I call the `RegisterTypes`:
```
    public static class PlatformInitializer
    {
        public static void RegisterTypes(IContainerRegistry containerRegistry)
        {
            containerRegistry.GetContainer().RegisterServiceTypes();
        }
    }
```
And than I have the ContainerUtility where I register the ServiceType:
```
    public static class ContainerUtility
    {
        public static void RegisterServiceTypes(this IUnityContainer container)
        {
            container.RegisterSingleton<IDictaVoxLocalStore, DictaVoxLocalStore>();
        }
    }
```
I tried to copy the Class for `GetContainer()` from Prism.Unity.Forms but the problem is that the container in `PlatformInitializer` class on app launch, the `containerRegistry` gets `Prism.DryIoc.DryIoc.ContainerExtension` but I need `Prism.Unity.Unity.ContainerExtension`. And than I figured out that I can only get this with the Prism.Unity.Forms package which is not available for .NET MAUI.
I need this because I use The Unity container Dependency Injection so I can populate this container.

Thanks in advance for your help

**Comment by dansiegel** [comment]
We MIGHT be able to support MAUI with the Unity Container if/when Unity.Container 6.0 is released. However we will be shipping the Prism 9.0 preview with DryIoc support for MAUI and Commercial Plus license holders will have access to also use Microsoft.Extensions.DependencyInjection as well.

What you are showing me however in your issue is that you have no actual need for the IUnityContainer as you can register a singleton with the IContainerRegistry in the same exact way you're registering the singleton with the IUnityContainer.
